### ISSUE TITLE
NullPointerException - __renderPage - buildDeckList

### ISSUE THREAD

**Issue by david-allison-1** [ankidroid/Anki-Android#6613]
https://couchdb.ankidroid.org/acralyzer/_design/acralyzer/index.html#/report-details/0ccae861-bb92-445e-aecb-799e9e58407b

```
java.lang.NullPointerException: Attempt to invoke virtual method 'boolean org.json.JSONObject.optBoolean(java.lang.String)' on a null object reference
at com.ichi2.anki.widgets.DeckAdapter.processNodes(DeckAdapter.java:6)
at com.ichi2.anki.widgets.DeckAdapter.processNodes(DeckAdapter.java:12)
at com.ichi2.anki.widgets.DeckAdapter.buildDeckList(DeckAdapter.java:5)
at com.ichi2.anki.DeckPicker.__renderPage(DeckPicker.java:3)
at com.ichi2.anki.DeckPicker$12.onPostExecute(DeckPicker.java:8)
at com.ichi2.async.CollectionTask$TaskListener.onPostExecute(CollectionTask.java:1)
at com.ichi2.async.CollectionTask.onPostExecute(CollectionTask.java:4)
at com.ichi2.async.CollectionTask.onPostExecute(CollectionTask.java:1)
at android.os.AsyncTask.finish(AsyncTask.java:755)
at android.os.AsyncTask.access$900(AsyncTask.java:192)
at android.os.AsyncTask$InternalHandler.handleMessage(AsyncTask.java:772)
at android.os.Handler.dispatchMessage(Handler.java:107)
at android.os.Looper.loop(Looper.java:224)
at android.app.ActivityThread.main(ActivityThread.java:7520)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:539)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:950)
```

**Comment by Arthur-Milchior** [comment]
I believe it's a problem related to `mNameMap`, accessed while it is not yet entirely computed.

**Comment by david-allison-1** [comment]
This is still an issue on 2.12beta4. I believe I'm in contact with a user with this problem, and will redirect them here if confirmed.

**Comment by david-allison-1** [comment]
This may be related to #6383

**Comment by david-allison-1** [comment]
Likely fixed in 2.12beta6, let's see how it goes
